float64[] joint_goal
---
bool successed
string text
